docs/mappings.md

```diff
@@ -0,0 +1,120 @@
+---
+layout: page
+title: Mapping Standards
+---
+
+<style>
+.small-text {
+    list-style: none;
+    font-size: 0.9em !important;
+}
+.list-separation > li {
+    margin-bottom: 0.5em;
+}
+</style>
+
+# Mapping Standards
+
+To ensure consistency for the Parchment mapping data, both to the benefit of reviewers and to consumers of the mappings,
+this page shows the mapping standards and guidelines. These standards are the basis used by reviewers when checking
+contributions.
+
+The Parchment mapping set is based off of the names provided by the obfuscation maps that Mojang publishes for all 
+versions since 1.14. These names are referred to by Parchment as **'official'** names, **'Mojang'** names, or 
+**'mojmap'** names, in contrast to the _'obfuscated'_ names found in the client and server JARs.
+
+As a general standard, use American English spelling for words. For example, prefer `color` to `colour`, or `armor` to
+`armour`.
+
+## Parameter Names
+
+{:.list-separation}
+
+1. **Names must only contain alphanumeric characters, and must begin with a lowercase letter.**
+    - Alphanumeric characters means the characters in the ranges `A-Z`, `a-z`, and `0-9`.
+    - {:.small-text} _Examples:_ `someVariable`, not `delta%` or `Delta`
+
+1. **Names must be in lower camel case.**
+    - Lower camel case is written by joining together words and capitalizing the first character of each word except 
+      the first.
+    - {:.small-text} _Examples:_ `lowerCamelCase`, `areaTransformer`, `generatedItem`
+
+1. **Names should be named based on the parameter types and context of use.**
+    - They should be verbose and use complete words. Avoid omitting essential information for the purpose of keeping the 
+      name compact.
+    - {:.small-text} _Examples:_ `BlockPos adjacentPos, BlockPos currentPos`, not `BlockPos pos1, BlockPos pos2`
+
+1. **Avoid abbreivations or acronyms.**
+    - _Exception:_ Common or well-known abbreviations can be used: `IO` (input-output), `NBT` ([named binary tag][nbt])
```

Comment:
Any good example of a common or well-known abbrevation to replace it? (Also, we should consider maintaining a page of these exceptions).

Reply:
`Id` for identifier which is what we use for resource location keys in registries.

Reply:
Thing is some abbreviations should be "fully qualified" for example TPS should be written as ticksPerSecond imo.
The main reason being TPS is not a commonly used (outside of minecraft specifically) abbreviations.

If we're talking stuff like IO, GUI or IP, then sure it can be abbreviated.
But stuff like TPS which is uncommon outside of minecraft, should be fully qualified so "lay-mens" can understand it.

Reply:
How should we resolve that then? Should we maintain a list of accepted shortened abbreviations and amend the standard to say 'only those from this page can be shortened'? Should we maintain the list but keep the wording the same (and link to the listing page, of course)?

Reply:
In general, we need to maintain a list of all decisions relating to standards we have enforced whether abbreviations or namings (e.g. `other` or `*Block/Item`). So, I would probably have a separate page that can be updated as needed to reflect on the fly decisions made by the standards team on how certain things should be named.

Reply:
In that case, we can leave this unchanged and have someone from Standards to draft up how such a page would be organized. Since the standards are under the purview of the Standards team and not under the review process for the actual mapping data, we can formulate a faster process for including data to that page to actually reflect decisions made on-the-fly.

Reply:
For transparency: to address the need for an on-the-fly-updatable page of standards/conventions relating to mappings, the wiki page [`Mapping Conventions`](https://github.com/ParchmentMC/Parchment/wiki/Mapping-Conventions) has been made on the Parchment repository.

The repository wiki is more ideal for this as opposed to hosting the list on the website, as it is quickly accessible and editable by the standards team (not needing a member of the website team to sign off and merge edits and additions).